src/com/sap/piper/cm/ChangeManagement.groovy

```diff
@@ -90,6 +90,25 @@ public class ChangeManagement implements Serializable {
         }
     }
 
+    def getTransportRequests(String changeId, String endpoint, String username, String password) {
```

Comment:
I would like to propose to have (a) test(s) in the corresponding test class ```test/groovy/com/sap/piper/cm/ChangeManagementTest.groovy```.

Furthermore I would like to propose to stick to that pattern that there is always exactly one class under tests. With the approach here we have two classes under test: the step and the corresponding util class. It is better to have tests for the utils class and additionally tests for the step. In the step the utils class is mocked since there is no need to test the utils class again.

An example:

* From the perspective of making call to the solman backend it is the duty of the cm utils class to translate parameters received via signature into a valid command line and to make the call. This needs to be asserted with tests.

* The duty of the step is to prepare the parameters (e.g. from config, from commit history) and to perform finally a call to a method of the utils class. This needs to be asserted in the test class for the step.

Having more than one class (e.g. step and utils class) under test we repeat testing the utils class in case we re-use the utils class methods from different steps (e.g. small toolbox steps vs convenience steps).